### PR TITLE
#155 Added bold lines on the 2nd screen

### PR DESCRIPTION
**Pull request by NikitaSukhanov**

### CODE REVIEW

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java [hunk 1]
```diff
@@ -135,11 +142,13 @@ private static void drawLeftField(
         final int leftFieldColumns = nonogram.getLeftFieldColumns();
         final float startX = nonogram.getMainFieldTopLeftX();
         final float startY = nonogram.getMainFieldTopLeftY();
+        final float linesWidth = cellSize / 15f + 1f;
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java]: It looks like a function

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java [hunk 2]
```diff
@@ -229,15 +244,66 @@ private static void drawTopField(
...
+    }
+
+    /**
+     * Function for drawing horizontal bold lines.
+     *
+     * @param background - rectangle, which will be filled with lines
+     * @param canvas     - canvas received in 'onDraw'
+     * @param nonogram   - nonogram to draw
+     */
+    private static void drawBoldLinesHorizontal(RectF background,
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java]: Check coding style

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java [hunk 3]
```diff
@@ -229,15 +244,66 @@ private static void drawTopField(
...
     }
 
+    /**
+     * Function for drawing vertical bold lines.
+     *
+     * @param background - rectangle, which will be filled with lines
+     * @param canvas     - canvas received in 'onDraw'
+     * @param nonogram   - nonogram to draw
+     */
+    private static void drawBoldLinesVertical(RectF background,
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java]: Check coding style

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java (hunk 2 above)
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java]: It looks like one function with `drawBoldLinesVertical`

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java [hunk 4]
```diff
@@ -203,6 +216,8 @@ private static void drawTopField(
         _backgroundRect.right =
                 _backgroundRect.left + nonogram.getTopFieldWidth();
         canvas.drawRect(_backgroundRect, _cellsPainter);
+        canvas.drawRect(_backgroundRect, _boldLinesPainter);
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/nonogramDrawable/NonogramDrawer.java]: Lines look like lines in `drawTopField`

**Review by maik-nack — APPROVED** [review]
Good job!

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Pull request merged**